ui/ts/pages/nodes.ts

```diff
@@ -282,9 +283,9 @@ module AdminViews {
         }
 
         return m(".page", [
-          m(".section.primary", [
-            m.component(NavigationBar, ctrl.TargetSet()),
-            m("h2", title)
+          m(".section.primary", m.component(Components.Topbar, {title: title})),
```

Comment:
Topbar is inside of .section.primary for some pages, but outside of it for other pages.  What is the consequence of that?